torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12265,6 +12272,15 @@ def ref_pairwise_distance(input1, input2):
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
            sample_inputs_func=sample_inputs_transpose_swapdims),
+    OpInfo('transpose_copy',
+           ref=lambda x, dim0, dim1: ref_transpose(x, dim0, dim1).copy(),
+           # transpose_copy does not support automatic differentiation for outputs with complex dtype
```

Comment:
Please add "transpose_copy" to `GRADIENT_IMPLEMENTED_FOR_COMPLEX` in `pytorch/tools/autograd/gen_variable_type.py`.

Reply:
Done.